cg/meta/workflow/fastq.py

```diff
@@ -147,22 +147,45 @@ def parse_header(line: str) -> dict:
 
         return fastq_meta
 
+    @staticmethod
+    def parse_nanopore_header(line: str) -> dict:
+        fastq_meta = {"lane": None, "flowcell": None}
```

Comment:
```suggestion
        header_dict = my_dict = dict(iter((value_pair.split("=") for value_pair in line.split(" ")[1:])))
        lane = header_dict["ch"]
        flowcell = header_dict["flow_cell_id"]
```
This should be safer than using magic numbers

Reply:
But does it work? There are not equal signs on every "section"

Reply:
> dict(iter((value_pair.split("=") for value_pair in nanopore_header.split(" ")[1:])))
> {'runid': '86134885e7ff0e1b44adfc2142bcac582dc34dd3', 'read': '12876', 'ch': '449', 'start_time': '2022-03-16T11:25:34.467266+00:00', 'flow_cell_id': 'FAR96930', 'protocol_group_id': 'CS24_NewprimerpoolTest_N1-24_O25-48', 'sample_id': 'CS24', 'barcode': 'barcode01', 'barcode_alias': 'barcode01', 'parent_read_id': 'c94862ea-e60d-4770-8870-fca0243e98d8'}
>